client/mirage/factories/project.js

```diff
@@ -136,15 +136,7 @@ export default Factory.extend({
     return faker.random.number();
   },
 
-  dcpCommunitydistrict() {
-    return faker.random.word();
-  },
-
-  dcpCommunitydistricts: 'BX08;SI01;SI02;SI03',
-
-  dcpValidatedcommunitydistricts() {
-    return faker.random.word();
-  },
+  dcpValidatedcommunitydistricts: 'X08;R01;R02;R03',
```

Comment:
Should we update this to use the new format? It seems like validated are comma separated? Or no?

Reply:
Yes, good catch! Should be comma-separated. I'll fix.